pkg/server/test/write.go

```diff
@@ -1334,20 +1431,24 @@ func TestWriteCommand(t *testing.T, datastore storage.OpenFGADatastore) {
 					[]*openfgav1.TupleKeyWithoutCondition{},
 					test.tuples,
 				)
-				require.NoError(err)
+				require.NoError(t, err)
 			}
 
-			cmd := commands.NewWriteCommand(datastore)
+			cmd := commands.NewWriteCommand(datastore,
+				commands.WithWriteCmdEnableConditions(test.enableConditions))
 			test.request.StoreId = store
 			if test.request.AuthorizationModelId == "" {
 				test.request.AuthorizationModelId = test.model.Id
 			}
 			resp, gotErr := cmd.Execute(ctx, test.request)
 
-			require.ErrorIs(gotErr, test.err)
+			if test.err != nil {
+				require.ErrorIs(t, gotErr, test.err)
+				require.ErrorContains(t, gotErr, test.err.Error())
+			}
 
 			if test.response != nil {
```

Comment:
By this point I assume `gotErr` is nil. Just to make sure we cover our bases, will you please make sure this is the case by adding an assertion for this case where `require.Nil(t, gotErr)`.